aws/resource_aws_ebs_encryption_by_default_test.go

```diff
@@ -0,0 +1,70 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSEBSEncryptionByDefault_basic(t *testing.T) {
+	resourceName := "aws_ebs_encryption_by_default.test"
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
If the `Delete` function is to remain empty, can you please add an explicit `CheckDestroy: nil,` to this `TestCase`?

Ideally though, this resource seems like it should enable encryption by default when its added and disable encryption when its removed. The `CheckDestroy` in that case would verify that encryption is disabled. 😄